### PR TITLE
Improve transactions logging

### PR DESCRIPTION
**Pull request by MaceWindu**
- add isolation level to logs
- add async suffix for async transaction operations

### CODE REVIEW

**Review by sdanyliv — APPROVED** [review]

**Comment by MaceWindu** [comment]
@viceroypenguin not sure... it will require addtional field on trace event object. I would prefere to leave it as is for now

File: Source/LinqToDB/Data/DataConnection.Async.cs [hunk 1]
```diff
@@ -299,7 +304,7 @@ public async Task DisposeAsync()
 					dataConnection.OnTraceConnection(new TraceInfo(dataConnection, TraceInfoStep.AfterExecute, traceOperation, true)
 					{
 						TraceLevel    = TraceLevel.Info,
-						CommandText   = commandText,
+						CommandText   = dataConnection.TraceSwitchConnection.TraceInfo ? sql : commandText?.Invoke(context),
```
Comment (viceroypenguin) [inline Source/LinqToDB/Data/DataConnection.Async.cs:307]: * We already know `dataConnection.TraceSwitchConnection.TraceInfo == true` due to the `if()`
* We know that when it is `true`, the value is already set in `sql`.

```suggestion
						CommandText   = sql,
```

File: Source/LinqToDB/Data/DataConnection.cs [hunk 2]
```diff
@@ -1639,12 +1640,12 @@ public virtual void RollbackTransaction()
...
 
-				if (TraceSwitchConnection.TraceInfo)
+				if (dataConnection.TraceSwitchConnection.TraceInfo)
 				{
-					OnTraceConnection(new TraceInfo(this, TraceInfoStep.AfterExecute, traceOperation, false)
+					dataConnection.OnTraceConnection(new TraceInfo(dataConnection, TraceInfoStep.AfterExecute, traceOperation, false)
 					{
 						TraceLevel    = TraceLevel.Info,
-						CommandText   = commandText,
+						CommandText   = dataConnection.TraceSwitchConnection.TraceInfo ? sql : commandText?.Invoke(context),
```
Comment (viceroypenguin) [inline Source/LinqToDB/Data/DataConnection.cs:1648]: * We already know `dataConnection.TraceSwitchConnection.TraceInfo == true` due to the `if()`
* We know that when it is `true`, the value is already set in `sql`.

```suggestion
						CommandText   = sql,
```

**Review by viceroypenguin — APPROVED** [review]

File: Source/LinqToDB/Data/DataConnection.Async.cs (hunk 1 above)
Comment (MaceWindu) [reply Source/LinqToDB/Data/DataConnection.Async.cs:307]: whoops, it should be in error report... will fix

**Pull request merged**